src/simply/simply_voice.c

```diff
@@ -0,0 +1,100 @@
+#include "simply_voice.h"
+
+#include "simply_msg.h"
+
+#include "simply.h"
+
+#include <pebble.h>
+
+typedef struct VoiceDataPacket VoiceDataPacket;
+
+struct __attribute__((__packed__)) VoiceDataPacket {
+  Packet packet;
+  int8_t status;
+  char result[SIMPLY_VOICE_BUFFER_LENGTH];
+};
+
+static SimplyVoice *s_voice;
+
+static bool send_voice_data(int status, char *transcription) {
+  VoiceDataPacket packet = {
+    .packet.type = CommandVoiceData,
+    .packet.length = sizeof(packet),
+    .status = (uint8_t) status,
+  };
+  snprintf(packet.result, sizeof(packet.result), "%s", transcription);
```

Comment:
Hmm, it would be better to allocate a packet that is the exact size of the string, such as
 ```
size_t packet_length = sizeof(VoiceDataPacket) + transcription_length + 1;
uint8_t buffer[packet_length];
VoiceDataPacket *packet = (VoiceDataPacket *)buffer;
*packet = (VoiceDataPacket) {
  .packet.type = CommandVoiceData,
  .packet.length = packet_length,
};
strcpy(packet->result, transcription);
```

`simply_msg_send_packet` will copy the packet over with a malloc, which is we use the stack as a buffer.

You would also change `char result[SIMPLY_VOICE_BUFFER_LENGTH];` to char result[];` in the struct definition since now it is variable size.

Bonus points if you use strncpy, but I'll accept it without it (too much to type here).